client/state/push-notifications/test/reducer.js

```diff
@@ -88,16 +84,12 @@ describe( 'settings reducer', () => {
 		const previousState = {
 			settings: {
 				enabled: false,
-				dismissedNotice: true,
-				dismissedNoticeAt: 1466067124796,
 			},
 		};
 		deepFreeze( previousState );
 		const newState = serialize( reducer, previousState ).root();
 
 		expect( newState.settings ).to.eql( {
```

Comment:
It shouldn't be hard to convert this test away from Chai.

Reply:
Sure thing, refactored in 6897bd5c2cc1853f8314866e16b31b604a406740.